### ISSUE TITLE
Tickers break on Android

### ISSUE THREAD

**Issue by creativecreatorormaybenot** [flutter/flutter#29940]
[Code as Gist](https://gist.github.com/creativecreatorormaybenot/7e43a2ba7d597b5c903fd162fc293011)

[Video demonstration on imgur](https://i.imgur.com/x12rVrf.mp4)

There are two things wrong here for me:

 1. If animations should be *stopped*, they should not start to go wild.

 2. I need a solution to **use** `AnimationController`'s or `Ticker'`s **anyway** even when this setting (Android: `Settings -> Accessibility -> Remove animations`) is enabled.

The video also does not do justice of showing just how broken the animations are once that setting is enabled.

**Comment by jonahwilliams** [comment]
I think the problem here is that you've implemented a repeating animation without an unbounded animation. So we attempt to speed up the duration. You can fix the behavior in this case by specifying an [`AnimationBehavior`](https://docs.flutter.dev/flutter/animation/AnimationBehavior-class.html) of preserve.

`_animationController = AnimationController(duration: Duration(seconds: 1), vsync: this, animationBehavior: AnimationBehavior.preserve)`

**Comment by jonahwilliams** [comment]
Another approach might be to instead default to preserve for all animation controllers and manually specify normal behavior for the specific widgets which are safe to be updated.

**Comment by creativecreatorormaybenot** [comment]
<sub>How did I write "brake" instead of "break", embarrasing</sub>
@jonahwilliams `AnimationController.preserve` will resolve my issue, this was just me failing to understand what it actually does. However, I still do not quite understand why the animations are speeded up instead of stopping or how I can specify some animations to do so.
